aws_analysis_tools/ec2_events/__init__.py

```diff
@@ -1 +0,0 @@
-from ec2_events import Application, main
```

Comment:
I kept this as for backward compatibility, but it has been a few versions since. Do you think we can delete this?